User Guide/Projects.md

```diff
@@ -835,7 +866,7 @@ You can refer for more details on mosquitto commands in on clicking the **[Link]
 
 ## My topics ##
 
-After creating a project, create your first topic. The **My topics** tab is only available for **Publisher**. This tab shows the topic that you have published for this project.
+After creating a project, you can create first topic. The **My topics** tab is only available for **Publisher**. This tab shows the topic that you have published for this project.
```

Comment:
"you can create first topic." -> "you can create your first topic."

Reply:
done